app/Contracts/Models/AbstractSizeVariantNamingStrategy.php

```diff
@@ -17,21 +15,9 @@ abstract class AbstractSizeVariantNamingStrategy
 	/**
 	 * The name of the Flysystem disk where images are stored.
```

Comment:
This comment no longer applies

Reply:
Fixed: [cd9419e](https://github.com/LycheeOrg/Lychee/pull/2384/commits/cd9419eee696ad56845446e01cbb42f9bedef162)